### PR TITLE
Init repo

### PR DESCRIPTION
**Pull request by martincivan**
- extract sources of SSO redirect script from  https://github.com/QualityUnit/LiveAgent-Apps
- add CI
- add Docker

Part of https://github.com/QualityUnit/infrastructure-issues/issues/1259

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+services:
+  php:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: dev
+    networks:
+      - internal
+    volumes:
+      - ./docker/php/conf.d:/usr/local/etc/php/conf.d
```
Comment (matuszeman) [inline docker-compose.yml:11]: overwriting `/usr/local/etc/php/conf.d` is not good as it "hides" any php extension configured via `docker-php-ext-install`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v3
+      - name: Cache PHP dependencies
+        uses: actions/cache@v3
+        with:
+          path: LiveAgent/server/vendor
```
Comment (matuszeman) [inline .github/workflows/ci.yml:13]: path

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+name: CI
+on:
+  push
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
```
[the remark this answers is not in the document]
Comment (martincivan) [reply .github/workflows/ci.yml:7]: I like running it like commands, as its very easy to replicate it locally - just copy-paste commands.

File: docker-compose.yml (hunk 1 above)
Comment (martincivan) [reply docker-compose.yml:11]: What do you propose?

**Pull request merged**